_docs/getting-started/centos.md

```diff
@@ -109,18 +109,10 @@ Now we need the Python dependencies. Note the troubleshooting info below... the
 ```
 pip install --user \
     future \
-    graphviz \
-    hypothesis \
-    jupyter \
-    matplotlib \
     numpy \
     protobuf \
```

Comment:
Do we still need this, since we build protobuf internally?

Reply:
Yes :( The python interface still needs it. It luckily does not need to correspond to the version we use internally though.